src/script/common/c_content.cpp

```diff
@@ -2528,6 +2528,9 @@ void push_collision_move_result(lua_State *L, const collisionMoveResult &res)
 			lua_setfield(L, -2, "object");
 		}
 
+		push_v3f(L, c.new_pos / BS);
+		lua_setfield(L, -2, "new_pos");
```

Comment:
need to change `lua_createtable` to 0, 6 above

Reply:
true